xml/System.IO.Pipelines/PipeWriter.xml

```diff
@@ -323,10 +323,19 @@ You must request a new buffer after calling <xref:System.IO.Pipelines.PipeWriter
         <Parameter Name="state" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="callback">To be added.</param>
-        <param name="state">To be added.</param>
+        <param name="callback">The callback to register.</param>
+        <param name="state">The state object to pass to <paramref name="callback" /> when it's invoked.</param>
         <summary>Registers a callback that gets executed when the <see cref="T:System.IO.Pipelines.PipeReader" /> side of the pipe is completed.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+> [!IMPORTANT]
+> `OnReaderCompleted` may not be invoked on all implementations of <xref:System.IO.Pipelines.PipeWriter>. This will be removed in a future release.
```

Comment:
```suggestion
> `OnReaderCompleted` may not be invoked on all implementations of <xref:System.IO.Pipelines.PipeWriter>. This limitation will be removed in a future release.
```

Reply:
See my comment for OnWriterCompleted.

Reply:
I think we should use "method" instead of "limitation" since the idea is to stop using that API altogether:

```suggestion
> `OnReaderCompleted` may not be invoked on all implementations of <xref:System.IO.Pipelines.PipeWriter>. This method will be removed in a future release.
```